misc/docker/base_images/debian_7_wheezy_32bit/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM obspy/base-images:debian_7_wheezy_32bit
+
+MAINTAINER Tobias Megies
+
+# Set the env variables to non-interactive
+ENV DEBIAN_FRONTEND=noninteractive DEBIAN_PRIORITY=critical DEBCONF_NOWARNINGS=yes
+
+# install packages to install obspy and build deb packages
+RUN apt-get update && apt-get upgrade -y || true
+RUN apt-get update && apt-get -y --no-install-recommends install \
+    debhelper \
+    devscripts \
+    equivs \
+    fakeroot \
+    gcc \
+    gfortran \
+    git \
+    help2man \
+    lintian \
+    locales \
+    lsb-release \
+    python \
+    python-decorator \
+    python-dev \
+    python-imaging \
```

Comment:
I think some tests might be executed differently (or even skipped) depending on whether it's installed or not..